### PR TITLE
[SYSSETUP] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
[SYSSETUP] Update Polish (pl-PL) translation

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

File: dll/win32/syssetup/lang/pl-PL.rc [hunk 1]
```diff
@@ -178,12 +178,12 @@ BEGIN
     IDS_COMPUTERSUBTITLE "Należy podać nazwę dla tego komputera oraz hasło administratora."
     IDS_LOCALETITLE "Opcje regionalne i językowe"
     IDS_LOCALESUBTITLE "System ReactOS można dostosować pod kątem języka i sposobu wyświetlania danych."
-    IDS_LOCALETEXT "The Standards and formats are set to %s, and the location is set to %s."
-    IDS_LAYOUTTEXT "Your default text input language and method is: %s keyboard layout"
+    IDS_LOCALETEXT "Standardy i formaty jest ustawione na %s, a lokalizacja jest ustawiona na %s."
```
Comment (Saibamen) [inline dll/win32/syssetup/lang/pl-PL.rc:181]:
```suggestion
    IDS_LOCALETEXT "Standardy i formaty są ustawione na %s, a lokalizacja jest ustawiona na %s."
```

File: dll/win32/syssetup/lang/pl-PL.rc [hunk 2]
```diff
@@ -178,12 +178,12 @@ BEGIN
...
+    IDS_LAYOUTTEXT "Twój język i metodę wprowadzania tekstu określa układ klawiatury: %s"
```
Comment (Saibamen) [inline dll/win32/syssetup/lang/pl-PL.rc:182]:
```suggestion
    IDS_LAYOUTTEXT "Domyślnym językiem i metodą wprowadzania tekstu jest układ klawiatury: %s"
```

File: dll/win32/syssetup/lang/pl-PL.rc [hunk 3]
```diff
@@ -178,12 +178,12 @@ BEGIN
...
     IDS_DATETIMETITLE "Data i godzina"
     IDS_DATETIMESUBTITLE "Ustaw prawidłową datę i godzinę."
-    IDS_PROCESSTITLE "Finishing the Installation"
-    IDS_PROCESSSUBTITLE "Please wait while Setup completes the final steps."
+    IDS_PROCESSTITLE "Zakończenie instalacji"
```
Comment (Saibamen) [inline dll/win32/syssetup/lang/pl-PL.rc:185]: Not sure
```suggestion
    IDS_PROCESSTITLE "Kończenie instalacji"
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Review by Saibamen — APPROVED** [review]

**Pull request merged**